package.json

```diff
@@ -149,6 +149,11 @@
                 "type": "string",
                 "description": "The source root on this machine that is the equivalent to the serverSourceRoot on the server."
               },
+              "pathMappings": {
+                "type": "object",
+                "default": {},
+                "description": "A mapping of multiple localSourceRoots on this machine to remote serverSourceRoots on the server."
```

Comment:
The description says this map is local -> server, but testproject/launch.json looks like the opposite